src/main/java/mate/academy/spring/service/mapper/UserMapper.java

```diff
@@ -11,6 +11,7 @@ public UserResponseDto mapToDto(User user) {
         UserResponseDto responseDto = new UserResponseDto();
         responseDto.setId(user.getId());
         responseDto.setEmail(user.getEmail());
+        responseDto.setRoles(user.getRoles());
```

Comment:
You should not directly expose entities through the API. Instead of setting the roles directly from user entity, map it to RoleDto.